pkg/util/cluster/cluster.go

```diff
@@ -448,35 +446,22 @@ func (c *Cluster) createCluster(ctx context.Context, vnetResourceGroup, clusterN
 		}
 
 		oc.Properties.WorkerProfiles[0].VMSize = api.VMSizeStandardD2sV3
-		ext := api.APIs[v20210901preview.APIVersion].OpenShiftClusterConverter().ToExternal(&oc)
-		data, err := json.Marshal(ext)
-		if err != nil {
-			return err
-		}
-
-		ocExt := mgmtredhatopenshift20210901preview.OpenShiftCluster{}
-		err = json.Unmarshal(data, &ocExt)
-		if err != nil {
-			return err
-		}
+	}
 
-		return c.openshiftclustersv20210901preview.CreateOrUpdateAndWait(ctx, vnetResourceGroup, clusterName, ocExt)
+	ext := api.APIs[v20210901preview.APIVersion].OpenShiftClusterConverter().ToExternal(&oc)
+	data, err := json.Marshal(ext)
+	if err != nil {
+		return err
+	}
 
-	} else {
-		ext := api.APIs[v20200430.APIVersion].OpenShiftClusterConverter().ToExternal(&oc)
-		data, err := json.Marshal(ext)
-		if err != nil {
-			return err
-		}
+	ocExt := mgmtredhatopenshift20210901preview.OpenShiftCluster{}
```

Comment:
Since `preview` is in the name, it makes me think we will have to be updating this again once it is released? should we create a story for that now?

Reply:
@s-amann API releases are a separate thing, done via a different process, it doesn't happen as frequently as RP releases.